src/platform/site-wide/cta-widget/index.js

```diff
@@ -9,7 +9,7 @@ import CallVBACenter from '../../static-data/CallVBACenter';
 import SubmitSignInForm from '../../static-data/SubmitSignInForm';
 
 import { toggleLoginModal } from '../user-nav/actions';
-import { logout, verify } from '../../user/authentication/utilities';
+import { logout, verify, mfa } from '../../user/authentication/utilities';
```

Comment:
sooooo since you are working in this file, can you clean up the imports? every instance of `../../../` can simply be removed (so `../../../platform/` would be simply `platform/`) and all instances of `../../` can become `platform/`. This is optional, but campsite rule and all... :-D

Reply:
Imports cleaned up.